bft/testers/sqlite/runner.py

```diff
@@ -54,17 +52,20 @@ def run_sql_case(self, case: Case, mapping: SqlMapping) -> SqlCaseResult:
             if mapping.aggregate:
                 arg_names = [arg_names[0]]
             joined_arg_names = ",".join(arg_names)
-            arg_vals = ",".join(
-                [
-                    "'" + literal_to_str(arg) + "'"
-                    if arg.type == "string" and arg.value is not None
-                    else literal_to_str(arg)
-                    for arg in case.args
-                ]
-            )
+            arg_vals = []
+            for arg in case.args:
+                if arg.type == "string" and arg.value is not None:
```

Comment:
This section might be better done as extracted as its own function.

Reply:
kept it in a separate extraction function, thanks for the suggestion.